src/utils/sources-tree/getURL.js

```diff
@@ -89,7 +89,7 @@ function _getURL(source: Source, defaultDomain: string): ParsedURL {
           filename,
           group: "file://"
         };
-      } else if (host === null) {
+      } else if (!host) {
```

Comment:
The `url` lib returned `null` when info didn't exist, whereas the native `URL` returns "".  TIL.